airbyte-integrations/connectors/source-stripe/source_stripe/schemas/payout_balance_transactions.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": true,
+  "type": ["object", "null"],
+  "properties": {
+    "payout": {
+      "type": ["null", "string"]
+    },
+    "fee": {
+      "type": ["null", "integer"]
+    },
+    "currency": {
+      "type": ["null", "string"]
+    },
+    "source": {
+      "type": ["null", "string"]
+    },
+    "fee_details": {
+      "type": ["null", "array"],
+      "items": {
+        "properties": {
+          "application": {
+            "type": ["null", "string"]
+          },
+          "type": {
+            "type": ["null", "string"]
+          },
+          "description": {
+            "type": ["null", "string"]
+          },
+          "amount": {
+            "type": ["null", "integer"]
+          },
+          "currency": {
+            "type": ["null", "string"]
+          }
+        },
+        "type": ["null", "object"]
+      }
+    },
+    "available_on": {
+      "type": ["null", "integer"]
+    },
+    "status": {
+      "type": ["null", "string"]
+    },
+    "description": {
+      "type": ["null", "string"]
+    },
+    "net": {
+      "type": ["null", "integer"]
+    },
+    "exchange_rate": {
+      "type": ["null", "number"]
+    },
+    "type": {
+      "type": ["null", "string"]
+    },
+    "sourced_transfers": {
```

Comment:
I don't see this field in the documentation or in the return value. Where does this come from?

Reply:
@maxi297 I don't know. I copied `airbyte/airbyte-integrations/connectors/source-stripe/source_stripe/schemas/shared/balance_transactions.json` to create the payout_balance_transaction json and just added `payout` and `updated` to it and didn't think to check anything else since payout balance transaction is really a grouped copy of balance transaction grouped on payout.

Reply:
Ok! [This](https://docs.stripe.com/upgrades#2017-01-27) probably answers the question. It was probably kept in `balance_transactions.json` because of legacy reasons but would remove it for the new stream.

Reply:
@maxi297 Ok I'll remove it now